Dockerfile

```diff
@@ -18,11 +18,13 @@ COPY . /init-code/
 WORKDIR /init-code/
 RUN /init-code/composer-install.sh \
 	&& php composer.phar --no-interaction install \
-	&& rm -f composer.phar \ 
+	&& rm -f composer.phar \
 	&& gem install travis \
+	# Install shell completion
+	&& echo "y" | travis \
```

Comment:
Aby sa to pri spusteni `travis` prikazu nepytalo `Shell completion not installed. Would you like to install it now? |y|`.